techniques/css/C35.html

```diff
@@ -2,90 +2,75 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
    <head>
       <meta charset="UTF-8" />
-      <title>C35: Allowing for text spacing without wrapping</title>
+      <title>C35: 折り返しのないテキストの間隔を開けられるようにする</title>
```

Comment:
```suggestion
      <title>C35: 折り返しのないテキストの間隔を見込む</title>
```

でどうでしょう（allow forについては別の箇所でもコメントしています）